examples/sandbox/src/App.js

```diff
@@ -238,6 +240,7 @@ export default function App() {
                     <Results
                       titleField="title"
                       urlField="nps_link"
+                      thumbnailField="image_url"
```

Comment:
You'll need to add support for the `thumbnailField` in the following files:

packages/react-search-ui/src/containers/Results.js
packages/react-search-ui/src/containers/Result.js